kubernetes/.swagger-codegen-ignore

```diff
@@ -1,8 +1,8 @@
 .gitignore
 git_push.sh
-# Remove this once swagger-codegen 2.2.3 is released and we update. 
+# Remove this once swagger-codegen 2.3.0 is released and we update. 
```

Comment:
What I meant was the comment is not useful anymore if we are using the latest swagger-codegen code to generate this. e.g. we don't need to wait for 2.3.0 to get this feature and we should not ignore any file here (as our change is already in HEAD).

I see all those files are commented out here anyway, so we don't ignore them in generation? Even ApiClient that we are manually editing bellow?